wikicurses/main.py

```diff
@@ -35,6 +37,111 @@ def tabComplete(text, matches):
     return match
 
 
+def showImages():
+    """
+    Launch program to display the images for the current article.
+
+    The program is specified in the config, along with it's args.
+    Otherwise, the default is `feh`.
+    """
+    if 'wikipedia.org' not in wiki.siteurl:
+        ex.notify('Cancelled: Only works on Wikipedia')
+        return
```

Comment:
Yes, using a try statement should be fine if there is not a better method.

What wikis are not working with it?